packages/components/src/components/ScalarFloating/useResizeWithTarget.ts

```diff
@@ -0,0 +1,35 @@
+import { type ComputedRef, type Ref, computed, ref, watch } from 'vue'
+
+type ResizeOptions = {
+  enabled?: Ref<boolean> | ComputedRef<boolean>
+}
+
+export function useResizeWithTarget(
+  target: Ref<Element | undefined>,
```

Comment:
Best to use `MayebRefOrGetter` in hooks for for flexible composition
This then requires `toValue(target)` when using the value.